proto/minder/v1/minder.proto

```diff
@@ -1197,9 +1207,22 @@ message ListEvaluationResultsRequest{
 }
 
 message ListEvaluationResultsResponse {
-    // status is the list of evaluation results.
-    // Results will typically be grouped by profile and project.
-    repeated RuleEvaluationStatus status = 1;
+    reserved 1; // deprecated status
+    reserved "status";
+
+    message EntityProfileEvaluationResults {
+        string profile = 1;
```

Comment:
Is this `profile_id` or `profile_name`, or `profile_maybe_one_or_the_other`?

IMO, it's most important that users can get from an EntityProfileEvaluationResult back to the profile that caused the evaluation, so it should line up with some ID that we can do a `Get` on.